### DISCUSSION TITLE
Hello friends

### DISCUSSION THREAD

**Discussion by Lalo1111111111111** [WalletConnect/walletconnect-monorepo#1127] [Developer Support] [answered]
Can I connect my dapps platform to a different wallet?  send my link and collect it to another new wallet with a different address.  I want to reconnect my dapps platform to another wallet

**Comment by hackyguru** [comment, marked as answer]
Sorry, connecting to a different wallet is not possible with WalletConnect. WalletConnect can only be used to connect to wallets that you own.
